cats/cats-core/src/main/java/com/netflix/spinnaker/cats/cache/CompositeCache.java

```diff
@@ -70,6 +70,21 @@ public CacheData get(String type, String id, CacheFilter cacheFilter) {
         return allItems.values();
     }
 
+    @Override
+    public Collection<String> existingIdentifiers(String type, Collection<String> ids) {
+        HashSet<String> identifiers = new HashSet<>(ids.size());
```

Comment:
wtb 2 space indent (or maybe cats is a 4 spacer and we should be consistent in this method)

Reply:
was consistent with the existing file, don't really want to push a reformat in this changeset

Reply:
that's fair ... it just stood out because the loop below is 2 space indented. meh.